Cannot find module '@primecms/field'

I'm following the contribution guide (https://docs.primecms.app/#/contributing) but can't start the project.

```
@primecms/ui: $ SKIP_PREFLIGHT_CHECK=true react-app-rewired start
@primecms/core: $ DEBUG=prime* ts-node src
@primecms/ui: Starting the development server...
@primecms/core: /home/Desktop/primecms/prime/node_modules/ts-node/src/index.ts:228
@primecms/core:     return new TSError(diagnosticText, diagnosticCodes)
@primecms/core:            ^
@primecms/core: TSError: ⨯ Unable to compile TypeScript:
@primecms/core: src/modules/external/index.ts(4,37): error TS2307: Cannot find module '@primecms/field'.
@primecms/core:     at createTSError (/home/Desktop/primecms/prime/node_modules/ts-node/src/index.ts:228:12)
@primecms/core:     at getOutput (/home/Desktop/primecms/prime/node_modules/ts-node/src/index.ts:334:40)
@primecms/core:     at Object.compile (/home/Desktop/primecms/prime/node_modules/ts-node/src/index.ts:367:11)
@primecms/core:     at Module.m._compile (/home/Desktop/primecms/prime/node_modules/ts-node/src/index.ts:413:43)
@primecms/core:     at Module._extensions..js (internal/modules/cjs/loader.js:785:10)
@primecms/core:     at Object.require.extensions.<computed> [as .ts] (/home/Desktop/primecms/prime/node_modules/ts-node/src/index.ts:416:12)
@primecms/core:     at Module.load (internal/modules/cjs/loader.js:641:32)
@primecms/core:     at Function.Module._load (internal/modules/cjs/loader.js:556:12)
@primecms/core:     at Module.require (internal/modules/cjs/loader.js:681:19)
@primecms/core:     at require (internal/modules/cjs/helpers.js:16:16)
@primecms/core: error Command failed with exit code 1.
@primecms/core: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

--- a/docs/contributing.md
+++ b/docs/contributing.md
@@ -18,7 +18,9 @@ All pull requests and issues are welcome
 
 - Install project dependencies: `yarn`
 - Link together all the packages: `yarn setup`
-- Watch the packages for changes and recompile: `yarn start`
+- Compile all the packages (**needed** once): `yarn compile`
+- Watch the **prime-core** and **prime-ui** packages for changes and recompile: `yarn start`
+- Reset the repository to a clean state and re-install everything: `yarn reset`
 
 ## Pull Requests
 

